DEVDOCS-4780: [update] Installing Stencil, update Windows instructions (#1870)
bigcommerce/dev-docs: docs/stencil-docs/installing-stencil-cli/installing-stencil.mdx

### Method 1: Install dependencies using Chocolatey

> If you're not comfortable manually installing and configuring Python and Node.js on Windows, or if you prefer an easy installation option, use the [Chocolatey package manager](https://chocolatey.org/install) to install Stencil CLI's dependencies. To do so, [open PowerShell](https://docs.microsoft.com/en-us/powershell/scripting/getting-started/starting-windows-powershell?view=powershell-6) as an administrator, and run the following commands:

Reviewer: ```suggestion
If you're not comfortable manually installing and configuring Python and Node.js on Windows, or if you prefer an easy installation option, use the [Chocolatey package manager](https://chocolatey.org/install) to install Stencil CLI's dependencies.  Python and C++ build tools are required to compile some dependencies, but you may already have them installed. The troubleshooting section contains information about which versions Stencil currently uses. ... (run into errors go there)

To install Stencil CLI's dependencies, [open PowerShell](https://docs.microsoft.com/en-us/powershell/scripting/getting-started/starting-windows-powershell?view=powershell-6) as an administrator, and run the following commands:

```

Author: I will trust the engineer's testing of the installation instructions and not add this clarification.

```shell showLineNumbers copy
# Install Chocolatey
iex ((New-Object System.Net.WebClient).DownloadString("https://chocolatey.org/install.ps1"))

# Install git if you don't have it
choco install git

> # Install nvm windows and stencil compatible node.js

Reviewer: ```suggestion
# Install nvm windows and stencil-compatible node.js

```

### Method 1: Install dependencies using Chocolatey

> If you're not comfortable manually installing and configuring Python and Node.js on Windows, or if you prefer an easy installation option, use the [Chocolatey package manager](https://chocolatey.org/install) to install Stencil CLI's dependencies. To do so, [open PowerShell](https://docs.microsoft.com/en-us/powershell/scripting/getting-started/starting-windows-powershell?view=powershell-6) as an administrator, and run the following commands:

Reviewer: ```suggestion
If you prefer an easy installation option, use the [Chocolatey package manager](https://chocolatey.org/install) to install Stencil CLI's dependencies. To do so, [open PowerShell](https://docs.microsoft.com/en-us/powershell/scripting/getting-started/starting-windows-powershell?view=powershell-6) as an administrator, and run the following commands:

```

choco install nvm; nvm install 18.15.0; nvm use 18.15.0

# Install Windows C++ Build Tools (also installs python2)
npm install -g windows-build-tools --vs2019

> # Tell npm to use python2

Reviewer: ```suggestion
# Tell npm to use python3

```

Set-ExecutionPolicy -ExecutionPolicy RemoteSigned
```

> 

Reviewer: ```suggestion

### Visual Studio not found error
If you receive the error "Could not find VS", update to the most current version of Stencil CLI and run it using the version of Node.js indicated in this article.

### Python npm configuration error
If you receive "Error: Could not find any Python installation to use", update to the most current version of Stencil CLI and run it using the version of Node.js indicated in this article.

```
